pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -119,7 +119,22 @@ func haveOverlap(a1, a2 []string) bool {
 
 func (pl *VolumeRestrictions) PreFilter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
 	if pl.enableReadWriteOncePod {
-		return nil, pl.isReadWriteOncePodAccessModeConflict(ctx, pod)
+		if status := pl.isReadWriteOncePodAccessModeConflict(ctx, pod); !status.IsSuccess() {
+			return nil, status
+		}
+	}
+	// if there is no volume needs to be checked in the filter, skip the filter directly
+	volumesCnt := 0
+	for i := range pod.Spec.Volumes {
+		v := &pod.Spec.Volumes[i]
+		// fast path if there is no conflict checking targets.
+		if v.GCEPersistentDisk == nil && v.AWSElasticBlockStore == nil && v.RBD == nil && v.ISCSI == nil {
+			continue
+		}
+		volumesCnt++
```

Comment:
We can `return nil, framework.NewStatus(framework.Success)` here, right? Probably like:

```go
	for i := range pod.Spec.Volumes {
		v := &pod.Spec.Volumes[i]
		// fast path if there is no conflict checking targets.
		if v.GCEPersistentDisk == nil && v.AWSElasticBlockStore == nil && v.RBD == nil && v.ISCSI == nil {
			continue
		}
		return nil, framework.NewStatus(framework.Success)
	}
	return nil, framework.NewStatus(framework.Skip)
```

Reply:
nice catch... It's right, we only care if this `volumesCnt >=1`,we don't care the actual number.